src/api/python/setup.py.in

```diff
@@ -67,7 +67,12 @@ else:
         'binding': False,
     }
 
-    extra_compile_args = ["-std=c++17", "-fno-var-tracking"]
+    extra_compile_args = [
+        "-std=c++17",
+        "-fno-var-tracking",
+        # Suppress deprecation warnings for deprecated members of OptionInfo
```

Comment:
Hm, until now we haven suppressed these warnings, but one may argue that this is the devs' business to take care of, not the users'. This in not only because of OptionInfo, is it?

Reply:
Yes, it is only because of OptionInfo. This case is special in the sense that we are still using OptionInfo, which contains deprecated members in its data structure. All other cases of deprecation warnings are related to functions that we no longer use.